ConfigSpace/conditions.pyx

```diff
@@ -545,41 +531,28 @@ cdef class InCondition(AbstractCondition):
         self.value = values
         self.vector_values = [self.parent._inverse_transform(value) for value in self.values]
 
-    def __richcmp__(self, other: Any, int op):
+    def __eq__(self, other: Any) -> bool:
```

Comment:
I'm currently wondering why we need this given that this is a subclass of the `AbstractCondition`